Source/Core/Core/ConfigManager.cpp

```diff
@@ -11,6 +11,7 @@
 #include <string>
 #include <string_view>
 #include <variant>
+#include <thread>
```

Comment:
Is this an intentional change? The file is otherwise unchanged, and could just be removed from the commit.